### DISCUSSION TITLE
Name not in sudoers list

### DISCUSSION THREAD

**Discussion by qradar** [Security-Onion-Solutions/securityonion#6134] [Unsupported Versions] [answered]
Can anyone help me with this: 
![Sec](https://user-images.githubusercontent.com/93916912/140732864-6bee407b-f5ec-47f2-bd32-e6a19a2809c6.JPG)

I just finished installing Security Onion but when I try to run these commands with the created user, I get this error. I tried searching and the official Security Onion document says that the created user should already be  in the sudoers list.

**Comment by Sahale** [comment]
I don't know if everything is OK with your installation, but you can add user to sudoers with `# usermod -aG wheel seconion` or `# usermod -aG sudo seconion`

**Reply by qradar** [reply]
Thanks for the reply. The installation gave an error at 95 percent. Same happened when I was setting SO at my home but that is working fine.

Is there anyway to start installation from 95 percent again because it took more than 4 hour to reach 95 percent :(

**Reply by Sahale** [reply]
As far as I know, no :(

**Comment by qradar** [comment, marked as answer]
Anyone please help. This is what I got after running # usermod -aG wheel seconion command.

![Sec2](https://user-images.githubusercontent.com/93916912/140866852-fd32c2db-e21e-4929-a5e2-cf00f5e435a0.JPG)

**Reply by Sahale** [reply]
Login as root firstly. Or execute `$ su -` and enter root password.
After that execute `# usermod ...`

**Reply by qradar** [reply]
It doesn't lets me switch to root. Anyways I am doing a fresh installation but when I select type as Linux then there is no option to select version as CentOS in VirtualBox.

What should I do ? Should I select Fedora and proceed ?

**Reply by dougburks** [reply]
The Guest OS setting in VirtualBox is most likely just a generic preset for CPU, RAM, etc. which you would need to change anyway to meet our hardware requirements:
https://docs.securityonion.net/en/2.3/hardware.html

If you have further questions or problems, please start a new discussion with appropriate title.
